game-core/src/main/java/games/strategy/triplea/ui/screen/TileManager.java

```diff
@@ -130,8 +130,7 @@ private void releaseLock() {
   public void createTiles(final Rectangle bounds) {
     acquireLock();
     try {
-      // create our tiles
-      tiles = new ArrayList<>();
+      tiles.clear();
```

Comment:
bit of a nit, but I'm curious what's the motivation for the `clear()` instead of the new call?
Looking at this, i wonder why `tiles` is not a local variable instead of having class scope to begin with. Failing that, object GC is very quick, the clear only reclaims the object references in our list, which is a small footprint on memory.

Reply:
Variable re-use can be considered a SRP violation as the variable does not have just one meaning, one data definition. In this case I do think it makes it a bit more harder to think about compared to just having a local variable. The variable having class scope is really at issue. That seems very strange and invites additional interaction with the variable and makes the state of the variable harder to understand. If we can move the variable from class scope to local, that would be a real win.